### PR TITLE
Feature/auto complete example

### PR DESCRIPTION
**Pull request by lievenvg**
## PR Checklist

This PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/forms/examples/forms/pages/demo/demo.page.html [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+
+<div class="u-margin-bottom col-md-6 col-xs-12">
+	<div class="a-input">
+		<div class="a-input__wrapper">
+			<aui-auto-complete
+			id="hero-names"
+			placeholder="Choose your hero…"
+			[(ngModel)]="selectedHero"
+			label="name"
+			value="''"
```
Comment (jsebrech) [inline packages/forms/examples/forms/pages/demo/demo.page.html]: if it's not being evaluated by [value], shouldn't this just be ""?

File: packages/forms/src/lib/auto-complete/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# @acpaas-ui/ngx-components/forms
+
+Description goes here.
```
Comment (jsebrech) [inline packages/forms/src/lib/auto-complete/README.md]: Title and description?

File: packages/forms/src/lib/auto-complete/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+| `@Input() remote: boolean;` | false | Enable remote search. |
+| `@Input() minCharacters: number;` | 0 | The minimal required characters to trigger a search. |
+| `@Input() mask: string;` | null | A mask pattern for the inputfield (see `MaskModule` for more info). |
+| `@Input() clearInvalid: boolean;` | false | Clear invalid searchstring values on blur (resets to modelvalue). |
+| `@Input() label: string;` | - | Provide a custom label prop for collections. This will be matched against for searching. |
+| `@Input() value: string;` | - | Provide a custom value prop for collections. This will be used as the actual value. |
+| `@Input() showAllByDefault: string;` | - | An attribute which can be set to true to show all items in `this.data` by default if no query is provided. This does need the `data`-attribute to have a correct value to work. |
+| `@Input() data: any[];` | [ ] | An array of filtered objects (use lable for this) or a flat array of filtered strings. Data is used to do basic filtering inside the component. It is also possible to do the filtering yourself and provide the results directly. |
```
Comment (jsebrech) [inline packages/forms/src/lib/auto-complete/README.md]: "(use the label and value properties)"

File: packages/forms/examples/forms/pages/demo/demo.page.html (hunk 1 above)
Comment (lievenvg) [reply packages/forms/examples/forms/pages/demo/demo.page.html]: True

File: packages/forms/src/lib/auto-complete/README.md (hunk 2 above)
Comment (lievenvg) [reply packages/forms/src/lib/auto-complete/README.md]: Woops...

**Pull request merged**